pom.xml

```diff
@@ -1414,6 +1414,23 @@
             <groupId>commons-logging</groupId>
             <artifactId>commons-logging</artifactId>
           </exclusion>
+          <!-- Hive 2.3.4 -->
+          <exclusion>
+            <groupId>org.eclipse.jetty.aggregate</groupId>
+            <artifactId>jetty-all</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.eclipse.jetty.orbit</groupId>
+            <artifactId>javax.servlet</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>com.github.joshelser</groupId>
+            <artifactId>dropwizard-metrics-hadoop-metrics2-reporter</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>*</artifactId>
+          </exclusion>
```

Comment:
Hey @wangyum, could you please add some detail comments here to explain why we want to exclude these dependencies? I can tell why we want to exclude them but it would be hard for other maintainers to figure out the intention in the future.

(In general, I'd say every single non-trivial dependency treatment, like exclusion and shading, should require a detail comment to explain the intention. Unfortunately, we are not doing this historically.)

Reply:
Thank you @liancheng. I added a comment for each exclusion.

Reply:
@liancheng did you mean comments in the POM file?

Reply:
I also added comments to the POM file.